features/context.feature

```diff
@@ -310,7 +310,7 @@ Feature: Context consistency
     Then it should fail with:
       """
       [Behat\Testwork\Suite\Exception\SuiteConfigurationException]
-        `contexts` setting of the "default" suite is expected to be an array, string given.
+        `contexts` setting of the "default" suite is expected to be an array, `string` given.
```

Comment:
this looks wrong to me. string is the type here, not the value the user entered. So IMO, it should not be enclosed in backticks